src/config/hacktoberfest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { registerAs } from "@nestjs/config";
+
+export const HacktoberfestConfig = registerAs("hacktoberfest", () => ({
+  year: String(process.env.HACKTOBERFEST_YEAR ?? "2023"),
```

Comment:
Refactor (non-blocking): Default to the current year instead of hard-coding it to `"2023"`.

```suggestion
  year: String(process.env.HACKTOBERFEST_YEAR ?? `${new Date().getFullYear()}`),
```

Reply:
Great suggestion!

> Also if you default to the current year, we don't even need to specify the environment variable atm.

True. The only thing would be it would look abit weird during the "current" year up to October since it'd be querying for data in the future that doesn't exist yet. So this will enable us to keep the previous year's data up on the dashboard before hand.

Example: once 2024 rolls around, if we don't specify a year in the env variables, all of January '24, Feburary '24, etc etc. will have empty data since it would look for data in the range of Oct '24.